.NET/Microsoft.Recognizers.Text.Number/Parsers/BaseNumberParser.cs

```diff
@@ -696,21 +696,21 @@ private bool SkipNonDecimalSeparator(char ch, int distance)
             return ch == Config.NonDecimalSeparatorChar && !(distance <= decimalLength && isMultiDecimalSeparatorCulture);
         }
 
-        private List<string> GetMatches(string input)
-        {
-            var successMatch = TextNumberRegex.Match(input);
-            var matchStrs = new List<string>();
-
-            // Store all match str.
-            while (successMatch.Success)
-            {
-                var matchStr = successMatch.Groups[0].Value;
-                matchStrs.Add(matchStr);
-                successMatch = successMatch.NextMatch();
-            }
-
-            return matchStrs;
-        }
+        // private List<string> GetMatches(string input)
+        // {
+        //    var successMatch = TextNumberRegex.Match(input);
+        //    var matchStrs = new List<string>();
+
+        // // Store all match str.
+        //    while (successMatch.Success)
+        //    {
+        //        var matchStr = successMatch.Groups[0].Value;
+        //        matchStrs.Add(matchStr);
+        //        successMatch = successMatch.NextMatch();
+        //    }
+
+        // return matchStrs;
+        // }

```

Comment:
Please remove commented code.

Reply:
For some reason I actually see those files in my branch, I don't really know why they appear as deleted in the first commit (matcher.py & abstract_matcher.py). I'll let that for now as it is, thank you for pointing that and for the feedback!

![image](https://user-images.githubusercontent.com/54330145/64701632-82172200-d47f-11e9-8154-18654ccaf9bc.png)